Mathlib/Analysis/Calculus/FDeriv/Norm.lean

```diff
@@ -0,0 +1,200 @@
+/-
+Copyright (c) 2024 Etienne Marion. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Etienne Marion
+-/
+import Mathlib.Analysis.Calculus.Deriv.Abs
+import Mathlib.Analysis.Calculus.Rademacher
+
+/-!
+# Differentiabilty of the norm in a real normed vector space
+
+This file provides basic results about the differentiability of the norm in a real vector space.
+Wee give some links between differentiability as `x` and `t • x`, and also prove
+that if the norm is differentiable at `x` then `fderiv ℝ (‖·‖) x x = ‖x‖` (`fderiv_norm_self`)
+and `‖fderiv ℝ (‖·‖) x‖ = 1` (`norm_fderiv_norm`).
+
```

Comment:
Could you expand a little bit this file-level docstring, mentioning notably also `fderiv_norm_smul` and the `ContDiffAt` results? A bullet list would probably be easier to read, by the way.